src/WebUI/Resources.js

```diff
@@ -66,5 +61,14 @@ define("Environments/Providers/Kubernetes/Resources", ["require", "exports"], fu
     exports.WorkloadText = "Workload";
     exports.WorkloadDetails = "{0} details";
     exports.PodStatusPending = "Pending";
-    exports.MoreImagesText = "{0} and {1} more"
+    exports.MoreImagesText = "{0} and {1} more";

```

Comment:
we also need a way for zero filtering results. would you add image for that also?

Reply:
@madhuv-msft for Services it is a straight-forward implementation. In case of workloads we deal with two independent components filtering over data. I'll file an issue and take it up as an independent implementation